src/index.js

```diff
@@ -166,9 +168,11 @@ export default class TimeAgo extends Component<DefaultProps, Props, void> {
       passDownProps.dateTime = (new Date(date)).toISOString()
     }
 
+    const nextFormatter = defaultFormatter.bind(null, value, unit, suffix, then);
```

Comment:
This creates a new function every time render runs. While I do like having the ability to just call `nextFormatter` without having to pass the arguments, it's not worth having this here for the 99% of the time `nextFormatter` is not used/called.